src/pages/event.js

```diff
@@ -134,34 +109,59 @@ export function Event() {
       });
       setTrackedEvent(event.id);
     }
-  }, [event, succeededLoadingEvent, trackedEvent, setTrackedEvent]);
+  }, [event, trackedEvent, setTrackedEvent]);
 
-  useEffect(() => {
-    // Get new batch of tokens
-    if (eventId) {
-      dispatch(
-        fetchEventPageData({
-          eventId,
-          first: FETCH_POAPS_LIMIT,
-          skip: FETCH_POAPS_LIMIT * pageIndex,
+  function getNextBatch(batchSize) {
```

Comment:
getTokensNextBatch perhaps? getNextBatch of what I questioned when I saw this. Specify a return type makes sense?

Reply:
This is js :( ahahah

Reply:
I change the function name for `getTokensNextBatch`